src/constants/action-types.js

```diff
@@ -4,9 +4,12 @@ export const TRACK_REQUEST_SUCCESS = "TRACK_REQUEST_SUCCESS";
 export const TRACK_REQUEST_FAILURE = "TRACK_REQUEST_FAILURE";
 export const TRACK_CHANGED = "TRACK_CHANGED";
 export const TRACK_UPDATED = "TRACK_UPDATED";
+export const TRACK_USER_REQUEST = "TRACK_ADDED_REQUEST";
```

Comment:
Can we align these to be the same string as const name please mate? e.g. both TRACK_USER_REQUEST